cli/dcoscli/node/main.py

```diff
@@ -3,19 +3,24 @@
 Usage:
     dcos node --info
     dcos node [--json]
+    dcos node log [--follow --lines=N --master --slave=<slave-id>]
```

Comment:
--master and --slave aren't actually mutually exclusive.  You can log both at the same time.  It's just you have to choose at least one.  I don't think there's any way to express this in docopt.